source/frontend/src/mocks/mockProjects.ts

```diff
@@ -0,0 +1,159 @@
+import { Api_AcquisitionFile } from 'models/api/AcquisitionFile';
+
+export const mockProjects = (
+  id = 1,
+  name = 'Test ACQ File',
+  rowVersion = 1,
+): Api_AcquisitionFile => ({
```

Comment:
hmm, if the function is labelled mockProjects I would expect an array of projects here not an acq file that has projects attached.

Reply:
I think the last update with the tests didn't get pushed :P